recipes/orographic-precipitation/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "orographic_precipitation" %}
+{% set version = "0.1rc0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "13559a0f324f9761ee2534430feb628f71416ff1c3b10c79d3452b2c276246b2"
```

```suggestion
  sha256: 13559a0f324f9761ee2534430feb628f71416ff1c3b10c79d3452b2c276246b2
```

Could you please remove the quotes from ``sha256``?

---

Yes, of course. I'm just starting to familiarise myself with yaml-files. This one came up when I executed:
`conda skeleton pypi your_package_name`

Should I do the same with the quotation marks in the `url` or should I leave them be?

---

We are developing a new tool with the intention to replace the current conda skeleton
it is not ready yet, I didn't generate version for it yet. But if you want to, you can give a try
https://github.com/marcelotrevisani/grayskull

And I thought I had fixed this problem regarding the quotes on ``sha256`` on conda skeleton
which version are you using of ``conda-build``?

the ``url`` is fine, you can let it there, there is no need to change, just if you want to

---

> We are developing a new tool with the intention to replace the current conda skeleton
> it is not ready yet, I didn't generate version for it yet. But if you want to, you can give a try
> https://github.com/marcelotrevisani/grayskull

Certainly, I want to give it a try. Do you mind giving me a rundown on how to use it? Do I create an instance from `AbstractRecipeModel` in my git directory, supply the package name and version number? Sorry for the inconvenience, this is still very new to me.

> And I thought I had fixed this problem regarding the quotes on `sha256` on conda skeleton
> which version are you using of `conda-build`?

`conda-build 3.18.9`
Maybe you did and it's simply an issue on my end.

> the `url` is fine, you can let it there, there is no need to change, just if you want to

Thank you for your help and the approval!